relay_rpc/src/jwt.rs

```diff
@@ -202,11 +202,11 @@ pub trait VerifyableClaims: Serialize + DeserializeOwned {
         let now = Utc::now().timestamp();
 
         if matches!(basic.exp, Some(exp) if now - time_leeway > exp) {
-            return Err(JwtError::Expired);
+            return Err(JwtError::Expired { expiration: basic.exp } );
         }
 
         if now + time_leeway < basic.iat {
-            return Err(JwtError::NotYetValid);
+            return Err(JwtError::NotYetValid { basic_iat: basic.iat, now_time_leeway: now + time_leeway });
```

Comment:
Is it worth logging `time_leeway` separately from `now`? Comment above says `time_leeway` is configurable